client/app/dedicatedCloud/datacenter/backup/enable/dedicatedCloud-datacenter-backup-enable.html

```diff
@@ -4,7 +4,7 @@
         <button type="button"
```

Comment:
I'm not sure about this, why do we manage a function for click event, and why it's not manage by the transition ?

It should be a data-sref no ? And not managed directly in the view ? The close button do always the same thing, right ?

Reply:
Indead all close button will make a `$state.go("^")`.
For the moment it will be complicated to have it global due to mix of `oui-modal` and `widzard`. 
In fact we should manage not only the close button but also cancel and validate buttons.
I will try to find a beautiful way for handling it :-)